Lib/ufo2ft/__init__.py

```diff
@@ -38,7 +38,7 @@ def compileOTF(ufo, outlineCompilerClass=OutlineOTFCompiler,
     otf = outlineCompiler.compile()
 
     if featureWriters is None:
-        featureWriters = (KernFeatureWriter(ufo), MarkFeatureWriter(ufo))
+        featureWriters = defaultFeatureWriters(ufo)
```

Comment:
instead of a `defaultFeatureWriters(ufo)` function, I'd prefer this to belong to the `featureCompilerClass`, either as class attribute containing a list of FeatureWriter classes (that are instantiated with a ufo if `featureWriters=None`), or as a `@staticmethod` that takes a ufo and return a list of feature writer instances. E.g. client may want to customize the feature writer options by subclassing the FeatureCompiler and overriding the `defaultFeatureWriters` method.

Reply:
I'd just have defaultFeatureWriters = (Kern, Mark) in the top level init. Do we need more complexity than that?

Reply:
Nevermind, that's ok too